app/Http/Controllers/EventController.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use DB;
+use Illuminate\Http\Request;
+use App\Http\Requests\CreateEventRequest;
+use App\Http\Requests\EditEventRequest;
+use App\Models\Event;
+
+class EventController extends Controller
+{
+    protected $event;
+
+    /**
+     *Constructor for EventController
+     *
+     * @param Event $event []
+     */
+    public function __construct(Event $event)
+    {
+        $this->event = $event;
+    }
+    
+    /**
+     *Running index page
+     *
+     * @return function [index]
+     */
+    public function index()
+    {
+         $select = [
+            'id',
+            'title',
+            'description',
+            'start_time',
+            'end_time',
+            'place'
+         ];
+           $data = $this->event->select($select);
+           $event = $data->orderBy('id', 'DESC')->paginate(10);
```

Comment:
[rank A] DESC is default for order by in system list, isn't it? Please define it as global constant